File: Tops Work/PYTHON/Assignments/Module – 2 (Fundamentals of python)/Q3.py
# Write a Python program to get the Fibonacci series of given range. 

def fib(n):
    a, b = 0, 1
    for _ in range(n):
        print(a, end=' ')
        a, b = b, a + b

num = int(input("Enter the number: "))
fib(num)

File: Tops Work/SE/Assignments/Module-4/MODULE 4.2 (Programing with C++)/Q.3.cpp
// Write a program to find the multiplication values and the cubic values using inline function

#include <iostream>
using namespace std;

inline int square(int x)
{
    return x * x;
}

inline int cube(int x)
{
    return x * x * x;
}

int main()
{
    int number;

    // Input a number from the user
    cout << "Enter a number: ";
    cin >> number;

    // Display the square and cube of the number using inline functions
    cout << "Square of " << number << " is: " << square(number) << endl;
    cout << "Cube of " << number << " is: " << cube(number) << endl;

    return 0;
}
